### ISSUE TITLE
JAWS bot is reporting smart glance highlight information after Command "'b' (virtual cursor active). Cannot manually reproduce.

### ISSUE THREAD

**Issue by LouisDo10** [w3c/aria-at-app#1504]
## Description of Behavior

JAWS bot is reporting smart glance highlight information after Command "'b' (virtual cursor active). Cannot manually reproduce.

## Test Setup

- Test File: [test-03-navForwardsToSwitchOffState-jaws.collected.html](https://aria-at.netlify.app/tests/switch/test-03-navForwardsToSwitchOffState-jaws.collected.html)
- AT: JAWS (Version 2025.2506.170)
- Browser: Chrome (Version 139)
- Navigate Forwards to a switch that is off
- Switch example, Test 1
- Command: `'b' (virtual cursor active)`
